oshi-core/src/main/java/oshi/software/os/OSProcess.java

```diff
@@ -95,6 +106,44 @@
         OTHER
     }
 
+    /**
+     * <p>
+     * Constructor for OSProcess.
+     * </p>
+     */
+    public OSProcess() {
+    }
+
+    /**
+     * <p>
+     * Constructor for OSProcess.
```

Comment:
Can we be a little more verbose with the javadoc here, stating that passing the PID will create an object populated with current stats?  You can even say it's equivalent to `{@link OperatingSystem.getProcess(processID)}`.